webdriver/tests/support/keys.py

```diff
@@ -243,7 +243,7 @@ class Keys(object):
         )),
         ("EQUALS", OrderedDict(
             [
-                ("code", ""),
+                ("code", "NumpadEqual"),
```

Comment:
What's me confusing here is the location of `0` below for this entry. This basically means that this is not part of the Numpad, because [Numpad has a location of `3`](https://w3c.github.io/webdriver/#dfn-key-location) as per spec. So why this `NumpadEqual` have a location of `0`? Isn't that the normal `Equal` key?

Reply:
The location should be 3 actually. Nice catch.

Reply:
I'm still not 100% sure if this is correct now. Note that we have two different `equal` keys on the keyboard. One for the left block and one for the Numpad. Why should that entry be for the Numpad? In Firefox it's currently setup to use `location=0`. What is Chrome currently using?

Given that the current location is `0` I believe we should keep it and use the `Equal` key from the standard location? The `Enter` key above also looks suspicious, and we probably have even more issues or missing keys as part of the spec?

Reply:
The location is most likely incorrect. If you look at Webdriver spec, `Equal` is next to the other Numpad characters.

I believe this all resulted from failed communication.

 - The code for `Equal` and `Pause` was never declared (most likely because no one cared).
 - Codes testing this naively tested for `=` and `Pause` without code.
 - 0 is the default location, so code testing this use `0` for the location.